internal/services/containers/container_connected_registry_resource_test.go

```diff
@@ -455,5 +455,5 @@ resource "azurerm_container_registry_token" "test" {
   resource_group_name     = azurerm_container_registry.test.resource_group_name
   scope_map_id            = azurerm_container_registry_scope_map.test.id
 }
-`, data.RandomInteger, data.Locations.Primary)
+`, data.RandomInteger, "east us")
```

Comment:
Rather than hard coding here, can you update the overrides for the testing framework in [this file](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/.teamcity/components/settings.kt)?

Reply:
Hi @jackofallops - Thanks for the code review! But the settings.kt file overrides for the whole rp, is it possible to only override test configuration for one resource?

Reply:
@ms-henglu - it should be fine to override one of the 3 locations to east us for this test?